### PR TITLE
Create igorslima.json

### PR DESCRIPTION
**Pull request by igorslima**

### CODE REVIEW

**Comment by zalew16** [comment]
:)

File: _pins/igorslima.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: YOUR-USERNAME
```
Comment (hectorsector) [inline _pins/igorslima.json]: @igorslima make sure you replace `YOUR-USERNAME` with your actual username. Let us know if we can be of help!

**Comment by crichID** [comment]
@T4selogistics please do not close issues or pull requests you did not create! This is a repository for people to learn so we appreciate you allowing each person to close or merge their own. Thank you.

**Comment by brianamarie** [comment]
Hi @igorslima! Since it's been a few days, I'm going to go ahead and make the necessary changes and merge your file into master. Good work! 🎉

**Pull request merged**